docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+GitHub. This will ensure a timely response.
+## Authentication issues
+
+Authentication is a complex topic. We would advise reading the general
+[Google Cloud Authentication documentation](https://cloud.google.com/docs/authentication)
+first in order to determine what authentication strategy is most suitable for
+your application. After that, the[.NET Authentication documentation]
+(https://developers.google.com/api-client-library/dotnet/guide/aaa_overview)
+should help you implement that strategy.
+
+If either it's unclear how to implement your chosen authentication strategy, or
+you have code which you expect to work but which doesn't, it's worth trying to
+isolate that aspect from any other application code. Creating a small console
+application which initializes appropriate credentials and then uses them in a
+single API call can help to make things much clearer.
+
+## Network/environmental issues
+
+Often users report issues which turn out to be due to unusual network
+environments (in particular proxies which may not fully implement HTTP/2.0) or
+other unexpected platform. Please review the [list of supported platforms]
+(platforms.md) before reporting an issue; while unsupported platforms *may*
+work in some situations, we cannot offer help if you run into problems.
+
+If a problem looks like it *may* be network-related, it's worth attempting to
+reproduce the issue in a different network environment. For example, if you're
+seeing a problem when running behind a corporate firewall, try to reproduce it
+when running on a different network or within a Google Cloud Platform VM or
+container. ([Google Cloud Functions](https://cloud.google.com/functions) are
+very simple to create and deploy, as a quick way of performing a single API
+call as a test.)
+
+## API-specific (server) issues
+
+Sometimes APIs don't behave as you might expect them to even when the client
+library is "working" perfectly, in terms of accurately providing your requests
+to the API server and returning the response to your code.
+
+We know that what ultimately matters is whether or not your application behaves
+as you want it to, but for most autogenerated libraries, the maintainers of the
+libraries don't know the details of how to use the APIs. Each API has its own
+dedicate support page with details of the best ways of getting support for that
+API, as well as troubleshooting tips and common error messages. See the
+[Speech-to-Text API support page]
+(https://cloud.google.com/speech-to-text/docs/support) for an example of this.
+
+If you're not sure whether the problem is caused by the client library or the
+Cloud API, there are a few ways you can test this:
+
+- If you can find the method you want to call in the "REST reference" part of
+  the API, you may be able to try to call the method directly in the browser.
+  For example, when calling the Text-to-Speech API's `ListVoices` method,
+  the "Try it!" button on [the REST voices.list reference page] allows you to
```

Comment:
Do we need to add the (url) part here? (I may be missing something here)

Reply:
Eek - not at all sure what happened here. Thanks, fixed.